docs/team/kumaran.md

```diff
@@ -1,6 +1,5 @@
 ---
-layout: page
-title: Kumaran's Project Portfolio Page
+layout: page title: Kumaran's Project Portfolio Page
```

Comment:
this wont work, need to revert back to original layout

Reply:
ok changed it!